src/common/syncjournaldb.cpp

```diff
@@ -909,7 +909,7 @@ bool SyncJournalDb::setFileRecord(const SyncJournalFileRecord &_record)
         }
     }
 
-    qCInfo(lcDb) << "Updating file record for path:" << record._path << "inode:" << record._inode
+    qCInfo(lcDb) << "Updating file record for path:" << QString::fromUtf8(record._path) << "inode:" << record._inode
```

Comment:
I think what you want is fromLocal8Bit? This is indeed what we should use for QByteArrays coming from the system before display.... that said I'm unsure, could be processed before and we already know it's utf8?

Reply:
@er-vin `fromLocal8Bit` gives garbage. This has been processed before, and yes we know it's UTF-8. It's coming from `QByteArray csync_file_stat_t::path` which was set to `QString::fromWCharArray(handle->ffd.cFileName).toUtf8` before.

Reply:
OK. So we're good with utf8 indeed.

Reply:
Note: I'm wondering if you might want to do the same with the `_e2eMangleName` below, this is also a path stored as QByteArray... you might want to check that too.